Reword warning for nearest-projection mapping

MakisH opened the pull request:
Closes #135.

I followed the easier approach for now: convert the error to a warning (when FSI + connectivity) and add a note in the documentation. I think it does not need to be more complicated, while configuring FSI with mesh connectivity is already a specific enough case for a warning.

The adapter now refers to the documentation via URL, which I think we should start doing more often. Even if the targets change eventually, a URL is better than looking manually through the documentation.

@davidscn please check. I have not yet checked what happens with solid OpenFOAM solvers that can write displacements.

TODO list:

- [x] I updated the documentation in `docs/`
- [x] I added a changelog entry in `changelog-entries/` (create directory if missing) -> not needed with the changes after review
- [ ] Squash before merging

davidscn commented on docs/config.md:
```diff
@@ -279,7 +279,7 @@ Data is obtained at the face centers, then interpolated to face nodes. Here, we
 
 {% note %}
-As already mentioned, the `Fluid` participant does not need to provide the mesh connectivity in case of a standard FSI. Therefore, the `Solid` participant needs to provide it and nothing special needs to be considered compared to other mapping methods. This implementation supports all CHT-related fields, which are mapped with a `consistent` constraint.
+As already mentioned, the `Fluid` participant does not need to provide the mesh connectivity in case of a standard FSI writing forces. It can, however, provide the mesh connectivity when writing stresses. Alternatively, the `Solid` participant can provide mesh connectivity on the displacements mesh, in which case nothing special needs to be considered in the adapter implementation compared to oth ...
```
No unfortunately not. We can provide the connectivity, but writing the data (e.g. `Stress`) won't be handled correctly. That's one of (my) bad design decisions, which requires to address this in each data field again. Right now, I can only think of scenarios where this behavior would either fallback to NN (such as for `Forces` etc.) and be ignored or would just be erroneous. Enabling it doesn't make sense in my opinion, unless we change the implementation.

MakisH replied:
How would you phrase this part of the documentation, then?

davidscn replied:
Correct would be: Mapping with connectivity information is not implemented for FSI and therefore, not supported. It's only supported for CHT related fields.

MakisH replied:
I updated the message and made it trigger an error again.

MakisH commented:
I updated the documentation according to the review (forgotten that yesterday) and deleted the changelog entry, as the change is now tiny. @davidscn back to you.
